test/controllers/jobs_controller_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
+
+class JobsControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @job = jobs(:one)
+  end
+
+  test "should get index" do
+    get jobs_url
+    assert_response :success
+  end
+
+  test "should get new" do
+    get new_job_url
+    assert_response :success
+  end
+
+  test "should create job" do
```

Comment:
I think it'd be good practice to start writing a comment for the test case. An example of a specific test case that the test would cover would do it.